tests/hook_main.py

```diff
@@ -2356,8 +2356,8 @@ def speaker_size_patch(transaction):
     :return:
     """
     with stash['app'].app_context():
-        EventImageSizeFactory()
-        SpeakerImageSizeFactory()
+        EventImageSizeFactory(type='event-image')
+        SpeakerImageSizeFactory(type='speaker-image')
```

Comment:
There should be automatic. You shouldn't have to specify that the type of SpeakerImageSizeFactory is speaker-image